config/default/app_settings.json

```diff
@@ -283,64 +228,48 @@
     ],
     "can_view_last_visited_date": [],
     "can_view_message_action": [
-      "national_admin",
-      "district_admin",
       "program_officer",
       "chw_supervisor",
       "chw"
     ],
     "can_view_messages": [
-      "national_admin",
-      "district_admin",
       "program_officer",
       "chw_supervisor",
       "chw"
     ],
     "can_view_messages_tab": [
-      "national_admin",
-      "district_admin",
       "program_officer",
       "chw_supervisor",
       "chw"
     ],
     "can_view_outgoing_messages": [
-      "national_admin",
       "program_officer"
     ],
     "can_view_reports": [
-      "national_admin",
-      "district_admin",
       "program_officer",
       "chw_supervisor",
       "chw"
     ],
     "can_view_reports_tab": [
-      "national_admin",
-      "district_admin",
       "program_officer",
       "chw_supervisor",
       "chw"
     ],
     "can_view_tasks": [
-      "district_admin",
       "chw_supervisor",
       "chw"
     ],
     "can_view_tasks_tab": [
-      "district_admin",
       "chw_supervisor",
       "chw"
     ],
     "can_view_unallocated_data_records": [
-      "national_admin",
-      "district_admin",
       "gateway",
       "program_officer",
       "chw_supervisor",
       "chw"
     ],
     "can_view_users": [
-      "national_admin",
       "program_officer"
     ],
     "can_write_wealth_quintiles": []
```

Comment:
How come you're editing this? Do those roles not exist in the default config?

Reply:
These roles aren't defined in CHT config. I've given feedback for binod to handle this in the base branch, so it will go away once I rebase.

Reply:
Awesome.